### PR TITLE
doc(prismic): added dispatcher query to routable types

### PR DESCRIPTION
**Pull request by mo-fc**

### CODE REVIEW

File: source/docs/prismic/routable-types.md [hunk 1]
```diff
@@ -60,7 +61,45 @@ export default {
...
+      ...CategoryFragment
+      ...CategoryCacheControlFragment
+    }
+    ... on CmsPage {
+      ...CmsPageFragment
+      ...CmsPageCacheControlFragment
+    }
++    ... on Album {
++      ...AlbumFragment
++      ...AlbumCacheControlFragment # optional check note below for link
```
Comment (dpobel) [inline source/docs/prismic/routable-types.md:96]: I think the content of those fragment needs to be details (especially the
AlbumFragment)

File: source/docs/prismic/routable-types.md [hunk 2]
```diff
@@ -60,7 +61,45 @@ export default {
...
+Note we added a path field resolver that appends the shop's baseUrl to the `url` field of our custom Prismic type.
+
+## Add GraphQL type to the dispatcher query
+
+Override `DispatcherQuery` if you havent already done so. You can find it under `src/web/theme/modules/Router/DispatcherQuery.gql`.
+
+Add a segment to `DispatcherQuery` that represents your newly created GraphQL type as noted below:
+
+```diff
+...
```
Comment (dpobel) [inline source/docs/prismic/routable-types.md:73]: would be nice to add the new required imports

File: source/docs/prismic/routable-types.md [hunk 3]
```diff
@@ -60,7 +61,45 @@ export default {
 +};
 ```
 
-Note we added a path field resolver that appends the shop's baseUrl  to the `url` field of our custom Prismic type.
+Note we added a path field resolver that appends the shop's baseUrl to the `url` field of our custom Prismic type.
+
+## Add GraphQL type to the dispatcher query
+
+Override `DispatcherQuery` if you havent already done so. You can find it under `src/web/theme/modules/Router/DispatcherQuery.gql`.
```
Comment (dpobel) [inline source/docs/prismic/routable-types.md:68]:
```suggestion
Override `DispatcherQuery` if you haven't already done so. You can find it under `src/web/theme/modules/Router/DispatcherQuery.gql`.
```

File: source/docs/prismic/routable-types.md [hunk 4]
```diff
@@ -60,7 +61,45 @@ export default {
...
+    }
++    ... on Album {
++      ...AlbumFragment
++      ...AlbumCacheControlFragment # optional check note below for link
++    }
+  }
+}
+```
+
+Note: for more related to setting up cache control check [the cache control and cdn documentation](/docs/advanced/performance/cache-control-and-cdn)
```
Comment (dpobel) [inline source/docs/prismic/routable-types.md:102]:
```suggestion
Note: for more related to setting up cache control check [the cache control and CDN documentation](/docs/advanced/performance/cache-control-and-cdn)
```

File: source/docs/prismic/routable-types.md [hunk 5]
```diff
@@ -60,7 +61,45 @@ export default {
...
 ```
 
-Note we added a path field resolver that appends the shop's baseUrl  to the `url` field of our custom Prismic type.
+Note we added a path field resolver that appends the shop's baseUrl to the `url` field of our custom Prismic type.
+
+## Add GraphQL type to the dispatcher query
+
+Override `DispatcherQuery` if you havent already done so. You can find it under `src/web/theme/modules/Router/DispatcherQuery.gql`.
+
+Add a segment to `DispatcherQuery` that represents your newly created GraphQL type as noted below:
```
Comment (real34) [inline source/docs/prismic/routable-types.md:70]:
```suggestion
Add an [inline fragment](https://graphql.org/learn/queries/#inline-fragments) to the `DispatcherQuery` for your newly created GraphQL type:
```

**Review by real34 — APPROVED** [review]
FYI: I wanted to add a link to our dispatcher documentation but detected that it was outdated. I created #223 to track this.

File: source/docs/prismic/routable-types.md (hunk 1 above)
Comment (mo-fc) [reply source/docs/prismic/routable-types.md:96]: the `AlbumFragment` sure but the `AlbumCacheControlFragment` i have a link to the documentation which should be enough

**Review by dpobel — APPROVED** [review]

**Pull request merged**